lib/simpleofflinecache.js

```diff
@@ -5,9 +5,32 @@ var cacheHelper = require('sw-cache-helper');
 var debug = 0 ? console.log.bind(console, '[SimpleOfflineCache]') :
  function(){};
 
-function SimpleOfflineCache(cacheName) {
+// Default Match options, not exposed.
```

Comment:
I'm starting to think this is not as simple so maybe it worths to be extracted from _serviceworkerware_ into its own repository.